README.md

```diff
@@ -102,12 +161,74 @@ These commands export information using the history archives. This allows users
 
 This command exports ledgers within the provided range. 
 
-### Stellar Core Commands
+<br>
+
+### **export_transactions**
+
+```bash
+> stellar-etl export_transactions --start-ledger 1000 \
+--end-ledger 500000 --output exported_transactions.txt
+```
+
+This command exports transactions within the provided range.
+
+<br>
+
+### **export_operations**
+
+```bash
+> stellar-etl export_operations --start-ledger 1000 \
+--end-ledger 500000 --output exported_operations.txt
+```
+
+This command exports operations within the provided range.
+
+<br>
+
+### **export_effects**
+
+```bash
+> stellar-etl export_effects --start-ledger 1000 \
+--end-ledger 500000 --output exported_effects.txt
+```
+
+This command exports effects within the provided range.
+
+<br>
+
+### **export_assets**
+```bash
+> stellar-etl export_assets \
+--start-ledger 1000 \
+--end-ledger 500000 --output exported_assets.txt
+```
+
+Exports the assets that are created from payment operations over a specified ledger range.
+
+<br>
+
+### **export_trades**
+```bash
+> stellar-etl export_trades \
+--start-ledger 1000 \
+--end-ledger 500000 --output exported_trades.txt
+```
+
+Exports trade data within the specified range to an output file
+
+<br>
+
+***
+
+## **Stellar Core Commands**
 
 These commands require a Stellar Core instance that is v15.0.0 or later. The commands use the Core instance to retrieve information about changes from the ledger. These changes can be in the form of accounts, offers, or trustlines.
 
 As the Stellar network grows, the Stellar Core instance has to catch up on an increasingly large amount of information. This catch-up process can add some overhead to the commands in this category. In order to avoid this overhead, run prefer processing larger ranges instead of many small ones, or use unbounded mode.
-#### export_ledger_entry_changes
+
+<br>
+
+### **export_ledger_entry_changes**
```

Comment:
nit: update below to just say "flags can filter which ledger entry types are exported" and remove the "three data type flags" that way as we add new ledger entry types we don't need to keep updating